tests/cython/backwardchainer/test_bc.py

```diff
@@ -69,6 +69,7 @@ def test_bc_deduction(self):
         self.assertTrue(resultAC.tv == AB.tv)
         self.assertEquals("A", resultAC.get_out()[0].name)
         self.assertEquals("C", resultAC.get_out()[1].name)
+        chainer = None
```

Comment:
this line looks like no-op

Reply:
I was hoping that this would cause the chainer destructor to run.  In python, how does one trigger destructors or finalizers?